src/dodal/devices/i09_1_shared/hard_undulator_functions.py

```diff
@@ -23,15 +23,15 @@
 
 
 async def get_hu_lut_as_dict(lut_path: str) -> dict:
```

Comment:
That too while we are at it
```suggestion
async def get_hu_lut_as_dict(lut_path: str) -> dict[int, np.ndarray]:
```